### PR TITLE
Document custom ssl certificate

### PR DESCRIPTION
**Pull request by simonrw**
Add documentation for supporting custom SSL certificates. This is needed for some users behind corporate networks with SSL intercepting proxy servers.

[Rendered](https://localstack-docs-preview-pr-404.surge.sh/references/custom-ssl-certificates/)

### CODE REVIEW

File: content/en/references/custom-ssl-certificates.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
+---
+title: Custom SSL certificates
```
Comment (alexrashed) [inline content/en/references/custom-ssl-certificates.md:2]: Maybe we should rename this to "TLS" (SSL isn't really used anymore, same for the rest of the changes).

File: content/en/references/init-hooks.md [hunk 2]
```diff
@@ -139,10 +139,12 @@ services:
     volumes:
       - "/path/to/init-aws.sh:/etc/localstack/init/ready.d/init-aws.sh"  # ready hook
       - "${LOCALSTACK_VOLUME_DIR:-./volume}:/var/lib/localstack"
-      - "/var/run/docker.sock:/var/run/docker.sock"
+      - "/vr/run/docker.sock:/var/run/docker.sock"
```
[the remark this answers is not in the document]
Comment (simonrw) [reply content/en/references/init-hooks.md:142]: :facepalm: thanks

File: content/en/references/custom-ssl-certificates.md (hunk 1 above)
Comment (simonrw) [reply content/en/references/custom-ssl-certificates.md:2]: I think SSL is still a common term for these but I've updated to TLS anyway. Thanks!

File: content/en/references/custom-ssl-certificates.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+There are three options when running LocalStack:
+
+1. [creating a custom Docker image]({{< ref "#creating-a-custom-docker-image" >}}),
+2. [using init hooks]({{< ref "#custom-ssl-certificates-with-init-hooks" >}}) or
+3. [when running in host mode]({{< ref "#custom-ssl-certificates-with-host-mode" >}}).
+
+They all can be summarised as:
+
+1. get your proxy's custom certificate into the system certificate store, and
+2. configure [`requests`](https://pypi.python.org/pypi/requests) to use the custom certificate.
```
[the remark this answers is not in the document]
Comment (simonrw) [reply content/en/references/custom-ssl-certificates.md:28]: Great, thanks

**Review by alexrashed — APPROVED** [review]
Awesome! Thanks for the nice guide! This will help quite a lot of people behind (corporate) proxies!
And the markdown style and the commits are great (one line per sentence in markdown, one commit per change)! 🚀

**Pull request merged**